CRVResponse/src/CrvPhotonGenerator_module.cc

```diff
@@ -345,15 +356,16 @@ namespace mu2e
               {
                 timeTmp = fmod(timeTmp,_microBunchPeriod);
                 //photons before the digitization start get removed except photons 
-                //in the first 55ns which get moved to the interval between the end of 
-                //the microbunch period and the digitization end
+                //in the first 55ns (digitization end - microbunch period) which get 
+                //moved to the interval between the end of the microbunch period and 
+                //the end of the digitization window.
                 if(timeTmp<digitizationEnd-_microBunchPeriod) timeTmp+=_microBunchPeriod;  
-                if(timeTmp<digitizationStart) continue;
+                if(timeTmp<digitizationStart-_digitizationStartMargin) continue;
```

Comment:
Does this part still work if digitizationEnd is > _microBunchPeriod?

Reply:
Yes, I tested it with a digitizationEnd of 1750ns.